apps/stable/metrics-server/values.yaml.gotmpl

```diff
@@ -0,0 +1,2 @@
+serviceAccount:
+  name: metrics-server
```

Comment:
I don't think we care about this. The default name will do. Please remove `apps/stable/metrics-server/values.yaml.gotmpl`